### PR TITLE
Update Participate & Learn Section

### PR DESCRIPTION
**Pull request by vipulgupta2048**
:hatching_chick: 
Finally updated #98

### CODE REVIEW

File: data/data-home.json [hunk 1]
```diff
@@ -96,7 +119,7 @@
             },
             {
                 "name":"Parth Sharma",
-                "designation":"Student",
+                "designation":"Student ex-VP",
```
Comment (thisisayush) [inline data/data-home.json]: Come on? @parthsharma2 seeing this?

**Comment by thisisayush** [comment]
And there a few conflicts too :P @vipulgupta2048

**Comment by vipulgupta2048** [comment]
Houston, we are ready for launch!

**Comment by vipulgupta2048** [comment]
Another change coming through, Shivam Bhaiya moved to Alumni

**Comment by thisisayush** [comment]
This is Houston, we're still seeing a few conflicts out there. Are you sure?

**Comment by vipulgupta2048** [comment]
Fixed, love the GitHub web editor for removing conflicts.

**Comment by thisisayush** [comment]
Reset any undesired changes to build.py and data-home.json, currently, it's showing changes in the whole file.

**Comment by vipulgupta2048** [comment]
I had pulled the wrong version apparently and merged it with my branch.

**Comment by thisisayush** [comment]
Any update?

**Comment by thisisayush** [comment]
Stop playing with Commit History maybe? Just look at the commits 😂

**Comment by thisisayush** [comment]
Squash it all up, and how did a travis commit came in here?

**Comment by vipulgupta2048** [comment]
So, 2 things 
Due to a massive amount of PR's being merged. It lead to this branch going down the pile of commits. I had to update it to stop conflicting with other files. 
Travis will make a commit on every deploy to keep track of the history. That's how I set it up. It ain't force pushing directly to the target branch.

**Comment by vipulgupta2048** [comment]
For instance check this out, 

```
d149683 (HEAD -> vipul-play, origin/vipul-play) Merge branch 'master' into vipul-play
249edae Merge pull request #197 from Nikhil-Soni-21/patch-1
b16a0ef Update data-home.json
4a62701 Merge pull request #184 from vipulgupta2048/update-webinars
e39f45d (update) Console Output Formatting for build script
bb3af45 (fix) Links not clickable in Communities Page
8aa4114 (fix) Contribution Guidelines in wrong file
9d4663b (update) Add virtual environments, pycache, temp and IDE files to gitignore
aded478 (update) Build Script Documentation and Linting
7677370 (update) Upgrade Requirements
3a7150d Merge pull request #188 from vipulgupta2048/master
f998867 Add Travis
9b2d285 Delete .travis.yml
ce6941c (origin/update-webinars, update-webinars) Merge branch 'master' into update-webinars
2e8dc12 Merge branch 'master' into vipul-play
f215b1b Merge pull request #181 from vipulgupta2048/vipul-fix-travis
392d3c9 Merge branch 'master' into vipul-play
e1fee1b Merge branch 'master' of https://github.com/vipulgupta2048/asetalias.github.io into vipul-play
**a6c194a Optimized VipulG and remove exVP**
**3ae9725 Optimized VipulG and remove exVP**
6bbaa13 Merge pull request #182 from vipulgupta2048/master
084c6c6 Merge pull request #185 from vipulgupta2048/contribute
db1a41f Merge pull request #183 from asetalias/vipul-pulls
51b6b50 Merge pull request #177 from vipulgupta2048/vipul-comm
680b3b6 (origin/contribute, contribute) Add Contributing
46d6c7c Merge branch 'master' of https://github.com/asetalias/asetalias.github.io
76d8ead (upstream/vipul-pulls, vipul-pulls) Changes made as requested
18935a1 Merge pull request #165 from vipulgupta2048/vipul-contacts
4a4de8e Changes made as suggested
881feaa Added AMA's, cos why not?
571d4d5 Web Dev webinar
e02f75e Add GSoC Webinar
ff05df3 Update pull_request_template.md
0293617 Update pull_request_template.md
83b2e52 Create CONTRIBUTING.md
323f669 Create pull_request_template.md
cca4663 Merge pull request #1 from vipulgupta2048/vipul-issues
7ab2ede Update issue templates
7df5964 (origin/vipul-fix-travis, vipul-fix-travis) Making Travis FINALLY work
b419b2a Making Travis work
0aaece8 Making Travis work
19f6085 Making Travis work
701004b Making Travis work
5cfba64 Making Travis works
2f5f7d6 Blacked Build.py
eeed41c Making Travis works
**7b28cd8 (Temporary) Making Travis Work**
c3ada8c Final build
**4bf4a1a Add Vipul image**
**f18879e Little (More) Housekeeping**
**05f1adc Little Housekeeping**
**3e3d93a Update Participate Section**
a6fcf7e Merge pull request #178 from vipulgupta2048/master
```

The commits are buried under a heap of commits from other PR's and your force commits to master won't let me squash them.

**Comment by thisisayush** [comment]
@vipulgupta2048 Congrats again. I just merged your previous PR.

**Comment by vipulgupta2048** [comment]
Good arguments always lead to postive results.
Now, merge this as well.

**Comment by vipulgupta2048** [comment]
@thisisayush GitHub automatically closed this of shame because I deleted all my commits through a wrongly done `git reset --hard`
Massive clusterfuck @ the first day of the decade.

**Comment by vipulgupta2048** [comment]
You would be happy since the next PR will have 2-3 commits at most instead of an ugly 13.

**Pull request closed without merging**